pkg/controller/template/render.go

```diff
@@ -777,6 +778,33 @@ func cloudPlatformIngressLoadBalancerIPs(cfg RenderConfig) (interface{}, error)
 	}
 }
 
+// cloudPlatformLBIPAvailable returns true when DNSType is set to `ClusterHosted`
```

Comment:
Curious, based on the comment I'd expect some check for clusterhosted in the function. I guess it's implicit since the service enablement is dependent on this field?

(I know we do the same elsewhere in the template rendering, so I'm fine with it as is)